src/app/_hooks/fcm/use-fcm.ts

```
@@ -2,40 +2,63 @@ import { getToken, onMessage } from "firebase/messaging";
 
-      const token = await getToken(messaging!, {
-        vapidKey: process.env.NEXT_PUBLIC_FIREBASE_VAPID_KEY!,
-        serviceWorkerRegistration: registration,
-      });
+      let token: string | null = null;
+      let attempt = 0;
+
+      while (!token && attempt < MAX_TOKEN_RETRY) {
+        try {
+          token = await getToken(messaging!, {
+            vapidKey: process.env.NEXT_PUBLIC_FIREBASE_VAPID_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)